js-miniapp-bridge/src/bridge.js

```diff
@@ -114,6 +114,31 @@ MiniAppBridge.requestPermission = function(permissionType) {
     );
   });
 };
+
+/**
+ Below code will override the navigator.geolocation.getCurrentPosition method for only iOS
+ */
+if (isPlatform.iOS()) {
+    navigator.geolocation.getCurrentPosition = function(success, error, options) {
+        return MiniAppBridge.exec(
+            "getCurrentPosition",
+            {locationOptions: options},
+            function(value) {
+                try {
+                    var parsedData = JSON.parse(value)
+                    success(parsedData)
+                } catch(error) {
+                    error({
+                        code: GeolocationPositionError.POSITION_UNAVAILABLE,
```

Comment:
I'm not sure about this. Can we call `GeolocationPositionError` directly like this here?
@ts-leojoseph-r @corycaywood

Reply:
@minh-rakuten . Yes we can -> https://w3c.github.io/geolocation-api/#position_error_interface